README.md

```diff
@@ -1,13 +1,19 @@
-[![crates.io](https://img.shields.io/crates/d/alloc-cortex-m.svg)](https://crates.io/crates/alloc-cortex-m)
-[![crates.io](https://img.shields.io/crates/v/alloc-cortex-m.svg)](https://crates.io/crates/alloc-cortex-m)
+[![crates.io](https://img.shields.io/crates/d/embedded-alloc.svg)](https://crates.io/crates/embedded-alloc)
+[![crates.io](https://img.shields.io/crates/v/embedded-alloc.svg)](https://crates.io/crates/embedded-alloc)
 
-# `alloc-cortex-m`
+# `embedded-alloc`
 
-> A heap allocator for Cortex-M processors
+> A heap allocator for embedded systems.
+
+Note that using this as your global allocator requires nightly Rust.
 
 This project is developed and maintained by the [Cortex-M team][team].
```

Comment:
Should this be moved to the HAL team?

Reply:
That might make sense if they agree, but I think we can do it as a separate PR.